design/src/rfcs/rfc0027_new_home_for_cred_cache_and_builder_api_consistency.md

```diff
@@ -0,0 +1,146 @@
+RFC: Finding New Home for Credential Types
+===================================================
+
+> Status: Implemented
```

Comment:
Can you update the status on the other RFC too?

Reply:
Addressed in 46bdb68.